gui/default/assets/lang/lang-en.json

```diff
@@ -374,11 +387,15 @@
    "You have no ignored folders.": "You have no ignored folders.",
    "You have unsaved changes. Do you really want to discard them?": "You have unsaved changes. Do you really want to discard them?",
    "You must keep at least one version.": "You must keep at least one version.",
+   "You should never add or change anything locally in a \"{%receiveEncrypted%}\" folder.": "You should never add or change anything locally in a \"{{receiveEncrypted}}\" folder.",
    "days": "days",
    "directories": "directories",
+   "enter password to encrypt data to be sent": "enter password to encrypt data to be sent",
```

Comment:
Strange that this starts with a lowercase.
Also, this password is used for decryption as well, so "encrypt data to be sent" is just part of the story?

"Enter encryption password" ?